siad/miner.go

```diff
@@ -5,40 +5,38 @@ import (
 	"net/http"
 )
 
-// minerStartHandler looks for a number of threads and then starts the miner
-// with that many threads.
+// minerStartHandler handles the api call that starts the miner.
 func (d *daemon) minerStartHandler(w http.ResponseWriter, req *http.Request) {
 	// Scan for the number of threads.
 	var threads int
 	_, err := fmt.Sscan(req.FormValue("threads"), &threads)
 	if err != nil {
-		writeError(w, "Malformed number of threads", 400)
+		writeError(w, "Malformed number of threads", http.StatusBadRequest)
 		return
 	}
 
 	d.miner.SetThreads(threads)
 	err = d.miner.StartMining()
 	if err != nil {
-		writeError(w, err.Error(), 500) // TODO: Need to verify that this is the proper error code to be returning.
+		writeError(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	writeSuccess(w)
 }
 
-// minerStatusHandler retruns a json struct containing the miner's status.
+// minerStatusHandler handles the api call that queries the miner's status.
 func (d *daemon) minerStatusHandler(w http.ResponseWriter, req *http.Request) {
 	mInfo, err := d.miner.Info()
 	if err != nil {
-		writeError(w, "Failed to encode status object", 500)
+		writeError(w, "Failed to encode status object", http.StatusInternalServerError)
 		return
 	}
 	writeJSON(w, mInfo)
 }
 
-// minerStopHandler stops the miner.
+// minerStopHandler handles the api call to stop the miner.
```

Comment:
I've been using lowercase everywhere. api, json, and go. I fixed the go to Go because go means multiple things, but it's pretty clear what api and json mean, and I find that have a giant string of capital words in the middle of a sentence can disrupt the flow. You're talking about the functionality of the listener and how it uses JSON to yadda yadda etc.

idk.